use-cases/data-export.md

```diff
@@ -0,0 +1,84 @@
+# Data export
+
+The _Data export_ feature of the **Mews Connector API** provides a streamlined method for retrieving bulk data. This feature is designed for users who require access to a substantial amount of data, such as [Order items](../operations/orderitems.md#order-item), without the hassle of making multiple paginated requests subject to timeouts and other constraints.
+
+### User scenarios
+
+- You are a Mews partner onboarding a new property and you need an initial data sync
+- You are a Mews Multi-Property user and want to retrieve data shared across multiple properties
+- You are using a newly added API operation and are loading data for the first time
+- You wish to perform a cross-data sanity check
+
+### User benefits
+
+- Streamlined bulk data export process
+- Reduced API request complexity and potential timeouts
+- Improved scalability for large-scale data retrieval
+- Faster property onboarding process
+- Potential for future expansion to support further data entities
+
+> ### Restricted!
+>
+> This feature is currently under beta test and is subject to change. If you would like to take advantage of the feature, contact us via [partnersuccess@mews.com](mailto://partnersuccess@mews.com).
+
+## Initiate the export
+
+To initiate a bulk data export, use the [Add export](../operations/exports.md#add-export) operation, specifying the data _Entity type_, e.g. `OrderItem`.
```

Comment:
suggestion(non-blocking): I think we should mention here that the Add export operation returns an export ID which needs to be used in the next step.

Reply:
yes ok